Sources/Suggestions/URLExtensions.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  URLExtensions.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public extension URL {
+
+    /// Creates a URL from a suggestion phrase.
```

Comment:
I moved this function here from macOS. I noticed that iOS had it’s own function very similar and thought the two could share the code since they have same requirements.